src/Model/GroundWaterFlow/submodules/sm-gwf-sfr-constant.f90

```diff
@@ -0,0 +1,45 @@
+submodule(SfrModule) sm_gwf_sfr_constant
+contains
+
+  !> @brief Method for solving for standard reaches
+    !!
+    !! Method to solve the continuity equation for a SFR package
+    !! reach using the standard approach.
+    !!
+  !<
+  module subroutine sfr_calc_constant(this, n, d1, hgwf, qgwf, qd)
+    ! -- dummy variables
```

Comment:
Minor annoying comment -- if we are ditching the "--" in comments, maybe start off fresh with these two new submodule files?